internal/app/cli/list_test.go

```diff
@@ -88,14 +89,44 @@ func TestList(t *testing.T) {
 		r.On("List", mock.Anything, sp).Return(listRes, nil).Once()
 
 		// when: list from the repo
-		err := List(context.Background(), repoSpec, sp)
+		err := List(context.Background(), repoSpec, sp, OutputFormatPlain)
 		stdout := getStdout()
 		stderr := getStderr()
 
 		// then: there is no error
 		assert.NoError(t, err)
 		// and then: stdout outputs the listable ThingModel
 		assert.Contains(t, stdout, listRes.Entries[0].Name)
+		assert.NotContains(t, stdout, "{")
```

Comment:
We should check that the ouptut is no json maybe by:
```
var actual any
err = json.Unmarshal([]byte(stdout), &actual)
assert.Error(t, err)
```

Reply:
fixed